typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -40,20 +39,36 @@ const warpRouteTokenBalance = new Gauge({
   ],
 });
 
+export function readWarpRouteConfig(filePath: string) {
+  const config = readYaml(filePath);
+  if (!config) throw new Error(`No warp config found at ${filePath}`);
+  const result = WarpRouteConfigSchema.safeParse(config);
+  if (!result.success) {
+    const firstIssue = result.error.issues[0];
+    throw new Error(
+      `Invalid warp config: ${firstIssue.path} => ${firstIssue.message}`,
```

Comment:
Wdyt about just print out the whole error object?a

Reply:
He got this from a the SDK's chain metadata utils which extract the first one to make it easier to surface to users (e.g. in explorer). But in this context I agree there's no reason to pull just the first.

Reply:
fixed!